### PR TITLE
[Merged by Bors] - chore: move more `assert_not_exists` up

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Fin/Basic.lean [hunk 1]
```diff
@@ -67,6 +67,7 @@ This file expands on the development in the core library.
 
 -/
 
+
```
Comment (grunweg) [inline Mathlib/Data/Fin/Basic.lean:70]: Can you revert this one?

File: Mathlib/Data/Fin/Basic.lean [hunk 2]
```diff
@@ -67,6 +67,7 @@ This file expands on the development in the core library.
```
Comment (grunweg) [inline Mathlib/Data/Fin/Basic.lean:67]:
```suggestion

File: Mathlib/CategoryTheory/Monoidal/Tor.lean [hunk 3]
```diff
@@ -22,6 +22,8 @@ Possibly it's best to axiomatize delta functors, and obtain a unique characteris
 -/
 
 
+assert_not_exists ModuleCat.abelian
```
Comment (grunweg) [inline Mathlib/CategoryTheory/Monoidal/Tor.lean:25]: I think this (renaming) change is in another PR as well. But I don't care strongly about separating these PRs.

File: Mathlib/Algebra/GroupWithZero/Conj.lean [hunk 4]
```diff
@@ -11,7 +11,6 @@ import Mathlib.Algebra.GroupWithZero.Units.Basic
 -/
 
 assert_not_exists Multiset
--- TODO
```
Comment (grunweg) [inline Mathlib/Algebra/GroupWithZero/Conj.lean:14]: IIRC, this change was tweaked in the last PR. Can you check/revert to that?

**Review by grunweg — APPROVED** [review]
Just four small comments.
maintainer delegate, i.e. feel free to maintainer merge on my behalf once you have considered my comments.

**Comment by semorrison** [comment]
bors d+

File: Mathlib/CategoryTheory/Monoidal/Tor.lean (hunk 3 above)
Comment (adomani) [reply Mathlib/CategoryTheory/Monoidal/Tor.lean:25]: I'll just leave it in: it has been wrong for a while now, so I don't think that it is especially important to track this!

**Comment by adomani** [comment]
bors merge

**Pull request closed without merging**